elements/lisk-transactions/src/utils/verify.ts

```diff
@@ -72,6 +72,25 @@ export const verifyAmountBalance = (
 	return undefined;
 };
 
+export const verifyMinRemainingBalance = (
+	id: string,
+	account: Account,
+	minRemainingBalance: bigint,
+): TransactionError | undefined => {
+	const balance = BigInt(account.balance);
```

Comment:
`account.balance` is always `bigint` now

Reply:
To keep the consistent implementation with `verifyAmountBalance` and `verifyBalance` I used `BigInt` here.

Reply:
it's probably better to make it consistently not use the `BigInt`